po/de.po

```diff
@@ -1494,9 +1493,8 @@ msgid "`%s' is not a valid key command"
 msgstr "`%s' ist kein gültiges Tastenkommando"
 
 #: src/keymap.cpp:744
-#, fuzzy, c-format
```

Comment:
I think `c-format` should've been kept, since the string contains `%s`.

Reply:
You're right, I just `dd`ed without paying attention.